Classes/GlobalStateExplorers/DatabaseBrowser/FLEXMultiColumnTableView.h

```diff
@@ -0,0 +1,47 @@
+//
+//  PTMultiColumnTableView.h
+//  PTMultiColumnTableViewDemo
+//
+//  Created by Peng Tao on 15/11/16.
+//  Copyright © 2015年 Peng Tao. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "FLEXTableColumnHeader.h"
+
+@class FLEXMultiColumnTableView;
+
+@protocol FLEXMultiColumnTableViewDelegate <NSObject>
+
+@optional
```

Comment:
Can we make these required instead of optional? That way we get more compile-time safety and don't need `respondsToSelector:` checks.

Small naming nits - I think these would be more consistent with Cocoa style patterns:
`-multiColumnTableView:didTapLabelWithText:`
`-multiColumnTableView:didTapHeaderWithText:sortType:`